articles/active-directory/develop/quickstart-v2-ios.md

```diff
@@ -88,6 +88,14 @@ In a terminal window, navigate to the folder with the downloaded code sample and
 >    ```swift
 >    let kClientID = "Enter_the_Application_Id_Here"
 >    ```
+> 1. Edit **ViewController.swift** and replace the line starting with 'let kAuthority' with the following code snippet:
+>    ```swift
+>    let kAuthority = "https://Enter_the_Cloud_Instance_Id_here/Enter_the_Tenant_Info_Here"
+>    ```
+> 1. Edit **ViewController.swift** and replace the line starting with 'let kGraphEndpoint' with the following code snippet:
+>    ```swift
+>    let kGraphEndpoint = "Enter_the_Graph_Endpoint_Here"
```

Comment:
The replacement for this token also includes a trailing slash, which differs from line 127 below. Please confirm this will not be a problem.

Reply:
@mstada, I will make needed updates to make sure that it won't be a problem.

Reply:
Done.

Reply:
@oldalton @archieag This token was renamed to `Enter_the_MS_Graph_Endpoint_Host_Here`. Please replace. Thanks!

Reply:
Done